orangecontrib/spectroscopy/widgets/owfft.py

```diff
@@ -458,32 +483,25 @@ def calculateFFT(self):
         phases = np.vstack(phases)
 
         self.phases_table = build_spec_table(wavenumbers, phases,
-                                             additional_table=self.data)
+                                            additional_table=self.data)
```

Comment:
Don't change the whitespace. Pylint will complain anyway :)

Reply:
I changed because the "Pylint" pointed in these lines hahaha

Reply:
Okay just do whatever pylint says :D

Reply:
Although I guess it's best not to change whitespace you didn't otherwise edit, it makes it hard to focus on the real changes :)